src/unix/system.c

```diff
@@ -307,8 +308,11 @@ Sys_Init
 */
 void Sys_Init(void)
 {
+    _Bool   malloc_flag;
```

Comment:
Shouldn't this be named `malloc_needed`?
Also, it should be initialized to 0/`false`.